packages/appium/lib/cli/extension-command.js

```diff
@@ -150,20 +151,29 @@ class ExtensionCommand {
       }
     });
 
-    const listData = /** @type {ExtensionListData} */ (exts);
+    /**
+     * Type guard to narrow "installed" extensions, which have more data
+     * @param {any} data
+     * @returns {data is InstalledExtensionListData<ExtType>}
+     */
+    const extIsInstalled = (data) => Boolean(data.installed);
 
     // if we're just getting the data, short circuit return here since we don't need to do any
     // formatting logic
     if (this.isJsonOutput) {
       return listData;
     }
 
+    if (verbose) {
+      this.log.log(inspect(listData, {colors: true, depth: null}));
```

Comment:
should verbose mode respect json output?

Reply:
hmm, yes, afaict `--json` returns the same data as `--verbose`, but it's just formatted differently.   maybe we don't need `--verbose` at all.  @jlipps